Model/Sales/Repository/PackageRepository.php

```diff
@@ -73,6 +73,21 @@ public function selectPackageType(array $products): string
         return $package;
     }
 
+
+    /**
+     * @param array $products
+     *
+     * @return bool
+     */
+    public function productWithoutDeliveryOptions(array $products): bool
+    {
+        foreach ($products as $product) {
+            $deliveryOptionsEnabled = $this->isDeliveryOptionsEnabled($product);
+        }
+
+        return $deliveryOptionsEnabled;
```

Comment:
Volgens mij krijg je in phpstorm hier een bericht dat deliveryOptionsEnabled niet herkend is.